test/k8sT/microscope.go

```diff
@@ -45,16 +47,21 @@ var _ = Describe(microscopeTestName, func() {
 	})
 
 	AfterFailed(func() {
-		kubectl.CiliumReport(helpers.KubeSystemNamespace,
-			"cilium endpoint list")
+		if microscopeCancel != nil {
```

Comment:
if you call microscopeCancel the information will dump into monitor.log. Don't print that info please.